daemon/container.go

```diff
@@ -1137,6 +1138,31 @@ func (container *Container) shouldRestart() bool {
 		(container.hostConfig.RestartPolicy.Name == "on-failure" && container.ExitCode != 0)
 }
 
+func (container *Container) mountVolumes() error {
```

Comment:
@jlhawn perhaps we could make container.Copy() call this func to avoid duplication of code?